pdf/src/generator.js

```diff
@@ -358,6 +360,20 @@ const getAcceptationStatus = params => {
   return {};
 };
 
+/**
+ * Flatten ecoOrganisme object and keep only relevant properties for the PDF
+ *
+ * @param params
+ * @returns object
+ */
+const getFlatEcoOrganisme = params => {
+  return params.ecoOrganisme.name
```

Comment:
Ça fait planter la génération de pdf pour des BSD's sans ecoOrganisme `Cannot read property 'name' of null`